.github/workflows/ci.yml

```diff
@@ -2,11 +2,21 @@ name: CI workflow
 on: [push, pull_request]
 jobs:
   test:
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
     strategy:
       matrix:
         node-version: [12.x, 14.x, 16.x]
-        os: [ubuntu-latest, windows-latest, macOS-latest]
```

Comment:
I assume we're ok to drop the multiple OS support here?

Reply:
Yes! I asked about it some messages above.

> > Got it!! Working on that now. One more question, though: the clean way to run Redis is through a service, but that will limit the CI to a Linux runner. Is that okay?
> 
> yes, you can see an example on `async-cache-dedupe` or `fastify-redis`